### ISSUE TITLE
Contacts can't be added[BUG]

### ISSUE THREAD

**Issue by umc1627** [nanu-c/axolotl#897]
### Description

contactlist is empty, contacts can't be added
received messages can't be recognized by sender

### Steps to Reproduce

1. open menu 
2. contacts : Info: contactlist is empty
3. plus : add contact
4. enter name and number , click add
5. no contact is saved

**Expected behavior:** a new contact is saved

**Actual behavior:** contactlist remains empty

#### Versions

Please provide the Version as written in `Settings->About Axolotl` ; Axolotl Beta 1.5.0

#### Device
fairphone 2 with ubuntutouch

#### Link to Debug Log
Please provide a link to debug from ~/.cache/upstart/application-click-textsecure.nanuc_textsecure*.log
Be careful it contains sensible data

**Comment by olof-nord** [comment]
In an effort to avoid duplicate issues: what is the difference between this and https://github.com/nanu-c/axolotl/issues/768? They look similar to me.

**Comment by olof-nord** [comment]
Or is this issue more like this one, where the main issue is to create a new contact from within the app itself, and not import it from a file? https://github.com/nanu-c/axolotl/issues/808

**Comment by umc1627** [comment]
yes, I meant the same as in #808 but I also can't import an vcf contact from a file.

How should I provide a Link to Debug Log?

**Comment by shulamy1** [comment]
it is the same in manjaro and arch

ezik

**Comment by FP3RT** [comment]
(re)installed Axoloti
restarted
- comm.error signal-servers <close>
Same bug here
- cannot manually create new contact
  - no error message
- import from contacts-app :
  - select in contact-app
  - return to Axolotl ... hangs ... "busy importing"
  - after some minutes : message: comm. error with Signal-servers.
  - after confirmation on warning screen ... hangs as before

v 1.6.0 
@ Ubuntu xenial 16.04  ; unity 8
@ FP3+
wifi internet access
internet access granted to Axoloti

**Comment by nanu-c** [comment]
this needs re-implementation in libsignal-service-rs first https://github.com/whisperfish/libsignal-service-rs/issues/200

**Comment by nanu-c** [comment]
Workaround is, to add the user via signal desktop
